src/Sulu/Component/Media/SmartContent/MediaDataProvider.php

```diff
@@ -45,7 +45,7 @@ public function __construct(
         ArraySerializerInterface $serializer,
         RequestStack $requestStack,
         ReferenceStoreInterface $referenceStore,
-        Security $security,
+        Security $security = null,
```

Comment:
optional parameters in the middle is not longer allowed: https://php.watch/versions/8.0/deprecate-required-param-after-optional

So we would need to add it to the end or use `?Security $security`

Reply:
Woah, that's a heavy change. Gonna fix it.